modules/nw-external-dns-operator-logs.adoc

```diff
@@ -15,3 +15,45 @@ You can view External DNS Operator logs by using the `oc logs` command.
 ----
 $ oc logs -n external-dns-operator deployment/external-dns-operator -c external-dns-operator
 ----
+
+== External DNS Operator domain name limitations
+
+External DNS Operator uses the TXT registry which follows the new format and adds the prefix for the TXT records. This reduces the maximum length of the domain name for the TXT records. A DNS record cannot be present without a corresponding TXT record, so the domain name of the DNS record must follow the same limit as the TXT records. For example, DNS record is `<domain-name-from-source>` and the TXT record is `external-dns-<record-type>-<domain-name-from-source>`. 
```

Comment:
Add comma: "...registry, which follows the new format..."

Reply:
Addressed